src/vm/winrttypenameconverter.h

```diff
@@ -90,7 +91,7 @@ public :
     // Get TypeHandle from a WinRT type name
     // Parse the WinRT type name in the form of WinRTType=TypeName[<WinRTType[, WinRTType, ...]>]
     //
-    static TypeHandle GetManagedTypeFromWinRTTypeName(LPCWSTR wszWinRTTypeName, bool *pbIsPrimitive);
+    static TypeHandle LoadManagedTypeForWinRTTypeName(LPCWSTR wszWinRTTypeName, bool *pbIsPrimitive, ICLRPrivBinder* loadBinder = nullptr);
```

Comment:
I think we need to be a bit more consistent with respect to default parameters. There are 3 functions here with very similar to argument lists, but only one has default parameter. There are other functions in this PR that have `nullptr` as a default and others that do not. Lets try to make them consistent across the feature at least. I am absolutely fine adding a `nullptr` in places and making a simply comment (i.e. `/*binder*/nullptr`).

Reply:
Also, the `bool *` in this signature is an out. Out params should always be at the end of the signature.